src/core/ReactContext.js

```diff
@@ -20,7 +20,6 @@ var emptyObject = require('emptyObject');
  * and is accessible via `this.context` on ReactCompositeComponents.
  */
 var ReactContext = {
-
```

Comment:
Introduced when I added the lifecycle stuff and then removed it.  Reverting the file.